android-bindings/src/bindings.rs

```diff
@@ -359,7 +360,7 @@ pub unsafe extern "C" fn Java_com_mobilecoin_lib_Amount_get_1bytes(
 #[no_mangle]
 pub unsafe extern "C" fn Java_com_mobilecoin_lib_Amount_finalize_1jni(env: JNIEnv, obj: JObject) {
```

Comment:
I feel like the function name should also change, but maybe in a followup PR. It would be suboptimal to have different names in Rust and Java.

Reply:
Yeah but that has to be an SDK-side change, I can't do that in this PR

Reply:
Maybe add a FIXME/TODO for renaming?

Reply:
i flagged this with SDK team, I'm going to let them figure out how best to track this follow on work.